models/res_partner.py

```diff
@@ -14,6 +14,8 @@ class Partner(models.Model):
     id_nat = fields.Char(string='Id. Nat.', help="Identification Nationale - xx-Yxxxx-YxxxxxY (where x = number and Y = capitalised letter)")
     nif = fields.Char(string='NIF', help="Numéro d'Identification Fiscale - YxxxxxxxY (where x = number and Y = capitalised letter)")
 
+    is_child_partner = fields.Boolean(string='Is Child Partner', compute='_compute_is_child_partner', store=True)
+
```

Comment:
@iambenkis  why not use the field "parent_id" ?  you're adding complexity with this computed field.